volts2

```diff
@@ -238,5 +238,13 @@
         "author": "Everblush",
         "description": "A Dark, Vibrant and Beatiful colorscheme",
         "meta": "https://raw.githubusercontent.com/Everblush/lapce/main/volt.toml"
+    },
+    {
+        name = "jeb-theme"
```

Comment:
They are not in json format.

Reply:
let me see

Reply:
Now that's it, sorry for mistakes, because is my first time 😢😢